src/util/compareThought.ts

```diff
@@ -52,6 +53,17 @@ export const compareStringsWithEmoji = (a: string, b: string) => {
   return aStartsWithEmoji && !bStartsWithEmoji ? -1 : bStartsWithEmoji && !aStartsWithEmoji ? 1 : 0
 }
 
+/** A comparator that sorts meta-attributes above emojis. */
+export const compareStringsWithMetaAttributes = (a: string, b: string) => {
+  const aStartsWithMetaAttributes = !!a.match(STARTS_WITH_META_ATTRIBUTES_REGEX)
```

Comment:
Use https://github.com/cybersemics/em/blob/dev/src/util/isAttribute.ts to check if a string is a meta attribute.